### PR TITLE
#66 Support comments when adding records

### PR DESCRIPTION
**Pull request by verscheures**
As discussed in issue #66

### CODE REVIEW

File: records.go [hunk 1]
```diff
@@ -165,19 +165,30 @@ const (
 	RRTypeWKS RRType = "WKS"
 )
 
+// Define a function to create a comment option
```
Comment (joeig) [inline records.go:168]: It's good practice to start the comment with the function name (`WithComments defines a function to ...`).

File: records.go [hunk 2]
```diff
@@ -165,19 +165,30 @@ const (
...
+func WithComments(comments ...Comment) func(*RRset) {
```
Comment (joeig) [inline records.go:169]: Can I ask you to add a small unit test for this function?

**Review by joeig — COMMENTED** [review]
Thanks. I have two minor comments.

File: records.go (hunk 1 above)
Comment (verscheures) [reply records.go]: The  comment has been corrected

File: records.go (hunk 2 above)
Comment (verscheures) [reply records.go:169]: I new test has been for this  function

**Comment by joeig** [comment]
:+1:

**Pull request merged**